tasks/shell_crmsh/crm-cib-constraint-ticket.yml

```diff
@@ -31,9 +43,18 @@
     cmd: |
```

Comment:
```suggestion
    cmd: >-
```

Reply:
@richm I am keeping pipe in configure commands as they can be multi-line like primitives and it works. I have removed it for one line commands above them.

Reply:
@richm I kept it because lot of configure tasks contain lot of operations and additional parameters so they are loaded with newline character \n. This is reason why I kept it with pipe. 

Situation is not requiring multi-line input for checks and deletions of course, only configure itself has lot of inputs.

All this code was retested many times and crmsh is able to take multi-line input correctly.